### PR TITLE
Apply yapf to wannierberri, fix all flake8 errors

### LINKED ISSUE #153: Undefined variable in __sym_wann.py
The variable `ir` is not defined. @Liu-Xiaoxiong could you fix this?

https://github.com/wannier-berri/wannier-berri/blob/696e849773be7e108a0dcc8f0c94fca319a6b135/wannierberri/__sym_wann.py#L262

### PR DESCRIPTION
**Pull request by jaemolihm**
Final fix of all flake8 errors. After this PR is merged, please make sure that all future PRs do not introduce any flake8 errors. In other words, all PRs should pass the "Lint / flake (pull_request)" check.

Could we make lint check in CI required in the PR setting?

Removed all flake8 warnings for source code! See the checkmark in CI.

Most changes are made automatically by yapf. I did few manual edits (remove unused imports, change ##### to #, remove or comment unused variables).

Please use squash merge.

### TODO
- [x] To be merged after #151.
- [x] Resolve #153

Closes #153

### CODE REVIEW

File: wannierberri/__sym_wann.py [hunk 1]
```diff
@@ -91,197 +111,210 @@ def __init__(self,num_wann=None,lattice=None,positions=None,atom_name=None,proj=
...
+                    match_index = atom_index
+                    vec_shift = np.array(
+                        np.round(new_atom - np.array(wann_atom_positions[match_index]), decimals=2), dtype=int)
                 else:
-                    if atom_index==self.num_wann_atom-1:
-                        assert atom_index != 0,'Error!!!!: no atom can match the new one Rvec = {}, atom_index = {}'.format(self.iRvec[ir],atom_index)
+                    if atom_index == self.num_wann_atom - 1:
+                        assert atom_index != 0
+                        # 'Error!!!!: no atom can match the new one Rvec = {}, atom_index = {}'.format(
+                        #     self.iRvec[ir], atom_index)
```
Comment (Liu-Xiaoxiong) [inline wannierberri/__sym_wann.py:295]:
```suggestion
                    if atom_index==self.num_wann_atom-1:
                        assert atom_index != 0,(
                            f'Error!!!!: no atom can match the new atom after symmetry operation {sym+1},\n'
                            +f'Before operation: atom {atomran} = {atom_position},\n'
                            +f'After operation: {atom_position},\n'
                            +f'All wann_atom: {wann_atom_positions}'
                            )
```
Could you please make this change to solve the undefined variable in __sym_wann.py

**Review by stepan-tsirkin — APPROVED** [review]
Great, thank you.

@jaemolihm , please squash-merge

afterwards I will make this check obligatory for a merge

**Comment by stepan-tsirkin** [comment]
@jaemolihm , sorry you can not merge because the test coverage is slightly below requirement. But I still can, using administrator privileges. If I squash-merge, all changes will be attributed to me.  If that is fine for you and you do not care github statistics, I can do it. 

Otherwise, you may add a small test to increase the coverage. e.g. we have a lot of lines that through errors, but never happen in tests.

**Comment by jaemolihm** [comment]
@stepan-tsirkin I cannot merge because the codecov checks which are required fails. Could you squash merge?

**Pull request merged**